packages/pages/src/util/yextrcContents.ts

```diff
@@ -11,21 +11,34 @@ const validUniverses = ["sandbox", "production", "sbx", "prod", "qa", "dev"];
  *
  * @public
  */
-export const parseYextrcContents = () => {
+export const parseYextrcContents = (scope: string | undefined = undefined) => {
   let accountId: string | undefined;
   let universe: string | undefined;
+  let shouldUpdate = false;
   try {
     const yextrcContents: string = fs.readFileSync(".yextrc", "utf8");
     const parsedContents = YAML.parse(yextrcContents);
-    if (
+    if (scope && parsedContents[scope]) {
+      const scopedContents = parsedContents[scope];
+      accountId = !isNaN(Number(scopedContents.accountId))
+        ? scopedContents.accountId
+        : undefined;
+      universe = validUniverses.includes(scopedContents.universe)
+        ? scopedContents.universe
+        : undefined;
+    } else if (scope) {
+      shouldUpdate = true;
+    } else if (
       !isNaN(Number(parsedContents.accountId)) &&
       validUniverses.includes(parsedContents.universe)
     ) {
       accountId = parsedContents.accountId;
       universe = parsedContents.universe;
+    } else {
+      shouldUpdate = true;
     }
   } catch (e: any) {
     // Return undefined for both fields if .yextrc cannot be read
   }
-  return { accountId, universe };
+  return { accountId, universe, shouldUpdate };
```

Comment:
I don't think this should return an update status. It should only return the accountId and universe given a scope. If those come back as undefined then we can add them.

Reply:
Removed.